apps/gro/fields/country.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+module.exports = {
+  'country': {
+    className: ['typeahead', 'js-hidden'],
+    options: [''].concat(require('../../../assets/countries').allCountries),
```

Comment:
Is this so the first option is blank? Maybe worth revisiting this in future - extending the mixin to accept a default value